benchmark/server/main.go

```diff
@@ -28,7 +29,8 @@ func main() {
 			grpclog.Fatalf("Failed to serve: %v", err)
 		}
 	}()
-	addr, stopper := benchmark.StartServer(benchmark.ServerInfo{Addr: ":0", Type: "protobuf"}) // listen on all interfaces
+	nw := &latency.Network{}
+	addr, stopper := benchmark.StartServer(nw, benchmark.ServerInfo{Addr: ":0", Type: "protobuf"}) // listen on all interfaces
```

Comment:
Pass a `nil` network instead.
If you move nw into `ServerInfo` as what I suggested in previous comment, this line doesn't need to be changed.